service-front/module/Application/src/Controller/IndexController.php

```diff
@@ -130,10 +130,16 @@ private function processLpaResponse(string $type, array $data): array
         throw new HttpException(400, 'Person type "' . $type . '" is not valid');
     }
 
-    public function abandonFlowAction(): ViewModel
+    public function saveCaseProgressAction(): ViewModel
```

Comment:
Should this not still be `abandonFlow`, since it's only used when you specifically abandon the journey?